src/java.base/share/classes/java/nio/file/Path.java

```diff
@@ -50,7 +50,7 @@
  * file system. {@code Path} defines the {@link #getFileName() getFileName},
  * {@link #getParent getParent}, {@link #getRoot getRoot}, and {@link #subpath
  * subpath} methods to access the path components or a subsequence of its name
- * elements.
+ * elements, and {@link #getExtension() getExtension} to obtain its extension.
```

Comment:
Could also mention the other new methods.

Reply:
Resolved in 030a250aaf4c309d515507527932b233f210501e.